src/Service.Host/client/src/components/SernosSequences.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Loading, EntityList, CreateButton } from '@linn-it/linn-form-components-library';
+import Page from '../containers/Page';
+
+function SernosSequences({ sernosSequences, loading }) {
+    return (
+        <Page>
+            {loading ? (
+                <Loading />
+            ) : (
+                <Fragment>
+                    <CreateButton createUrl="/products/maint/sernos-sequences/create" />
+                    <EntityList
+                        title="Sernos Sequences"
+                        entityList={sernosSequences}
```

Comment:
are sernos sequences displayed in a meaningful sequence (order)? e.g. alphabetical or by id or whatever.